platform/darwin/test/MGLDocumentationExampleTests.swift

```diff
@@ -103,6 +103,18 @@ class MGLDocumentationExampleTests: XCTestCase, MGLMapViewDelegate {
 
         XCTAssertNotNil(mapView.style?.source(withIdentifier: "pois"))
     }
+    
+    func testMGLPolyline() {
+        //#-example-code
+        let coordinates = [
+            CLLocationCoordinate2D(latitude: 35.68476, longitude: -220.24257),
+            CLLocationCoordinate2D(latitude: 37.78428, longitude: -122.41310)
+        ]
+        let polyline = MGLPolyline(coordinates: &coordinates, count: UInt(coordinates.count))
+        //#-end-example-code
+        
+        XCTAssertNotNil(mapView.annotations)
```

Comment:
This line is the test part of this test — currently, it’s checking to see if any annotations have been added to the map. I would expect this to fail, because `testMGLPolyline()` creates an MGLPolyline but does not add it (as an annotation or otherwise).

To make this test functional, maybe try checking some aspect of the `polyline` object.